Arelle/Arelle
# Make pip installed plugins show up in GUI

#### Reason for change
Closes #786 

#### Description of change
Reuse the plugin entry point discoverability implemented for CLI, for the GUI.
`CntlrWinMain` now calls a `PluginManager` function that discovers and configures any plugin modules installed in the environment.

#### Steps to Test
- Run GUI locally *without* `ixbrl-viewer` plugin pip-installed (or an external plugin of your own choosing)
- Confirm `ixbrl-viewer` not loaded in GUI
- `pip install ixbrl-viewer`
- Run GUI and confirm ixbrl-viewer is loaded in GUI

**review**:
@Arelle/arelle
